From: Simon McVittie <smcv@collabora.com>
Date: Tue, 31 Oct 2017 15:21:37 +0000
Subject: userns-block-fd: Search the PATH for bwrap

This means we can use it with an installed bwrap, which seems a more
common use of a demo script than a just-compiled bwrap, and is
consistent with the shell scripts.

Signed-off-by: Simon McVittie <smcv@collabora.com>

Closes: #246
Approved by: giuseppe
Applied-upstream: 0.2.1, commit:b8fa270e8937ed26e4e684050d221202b475c708
---
 demos/userns-block-fd.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/demos/userns-block-fd.py b/demos/userns-block-fd.py
index a390cce..67b6d8b 100755
--- a/demos/userns-block-fd.py
+++ b/demos/userns-block-fd.py
@@ -33,4 +33,4 @@ else:
             "--bind", "/", "/",
             "cat", "/proc/self/uid_map"]
 
-    os.execl(*args)
+    os.execlp(*args)
